NullPointerException in CommandParsers

```
[ERROR] [03/08/2014 21:49:11.782] [CodingteamSystem-akka.actor.default-dispatcher-114] [akka://CodingteamSystem/user/core] null
java.lang.NullPointerException
	at ru.org.codingteam.horta.core.CommandParsers$$anon$1.apply(CommandParsers.scala:13)
	at scala.util.parsing.combinator.Parsers$Parser$$anonfun$map$1.apply(Parsers.scala:242)
	at scala.util.parsing.combinator.Parsers$Parser$$anonfun$map$1.apply(Parsers.scala:242)
	at scala.util.parsing.combinator.Parsers$$anon$3.apply(Parsers.scala:222)
	at scala.util.parsing.combinator.Parsers$Parser$$anonfun$flatMap$1.apply(Parsers.scala:239)
	at scala.util.parsing.combinator.Parsers$Parser$$anonfun$flatMap$1.apply(Parsers.scala:239)
	at scala.util.parsing.combinator.Parsers$$anon$3.apply(Parsers.scala:222)
	at scala.util.parsing.combinator.Parsers$Parser$$anonfun$map$1.apply(Parsers.scala:242)
	at scala.util.parsing.combinator.Parsers$Parser$$anonfun$map$1.apply(Parsers.scala:242)
	at scala.util.parsing.combinator.Parsers$$anon$3.apply(Parsers.scala:222)
	at scala.util.parsing.combinator.RegexParsers$class.parse(RegexParsers.scala:148)
	at ru.org.codingteam.horta.core.SlashParsers$.parse(CommandParsers.scala:31)
	at ru.org.codingteam.horta.core.Core$$anonfun$ru$org$codingteam$horta$core$Core$$parseCommand$1.apply(Core.scala:101)
	at ru.org.codingteam.horta.core.Core$$anonfun$ru$org$codingteam$horta$core$Core$$parseCommand$1.apply(Core.scala:100)
	at scala.collection.immutable.List.foreach(List.scala:318)
	at ru.org.codingteam.horta.core.Core.ru$org$codingteam$horta$core$Core$$parseCommand(Core.scala:100)
	at ru.org.codingteam.horta.core.Core$$anonfun$receive$1.applyOrElse(Core.scala:63)
	at akka.actor.ActorCell.receiveMessage(ActorCell.scala:498)
	at akka.actor.ActorCell.invoke(ActorCell.scala:456)
	at akka.dispatch.Mailbox.processMailbox(Mailbox.scala:237)
	at akka.dispatch.Mailbox.run(Mailbox.scala:219)
	at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(AbstractDispatcher.scala:386)
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)
```

--- a/src/main/scala/ru/org/codingteam/horta/protocol/jabber/PrivateMucMessageHandler.scala
+++ b/src/main/scala/ru/org/codingteam/horta/protocol/jabber/PrivateMucMessageHandler.scala
@@ -24,8 +24,10 @@ class PrivateMucMessageHandler(muc: ActorRef, nick: String, implicit val executo
     case UserMessage(message) =>
       val jid = message.getFrom
       val text = message.getBody
-      (muc ? GetCredential(jid)) onSuccess {
-        case roomCred: Credential => core ! CoreMessage(Clock.now, roomCred.copy(location = self), text)
+      if (text != null) {
+        (muc ? GetCredential(jid)) onSuccess {
+          case roomCred: Credential => core ! CoreMessage(Clock.now, roomCred.copy(location = self), text)
+        }
       }
   }
 }

